packages/shared-core/src/constants/index.ts

```diff
@@ -98,6 +98,7 @@ export enum BuilderSocketEvent {
 export const SocketSessionTTL = 60
 export const ValidQueryNameRegex = /^[^()]*$/
 export const ValidColumnNameRegex = /^[_a-zA-Z0-9\s]*$/g
+export const ValidSnippetNameRegex = /^[a-z-_][a-z0-9-_]*$/i
```

Comment:
I've just realised allowing `-` is incorrect, so this should be:

```suggestion
export const ValidSnippetNameRegex = /^[a-z_][a-z0-9_]*$/i
```

Reply:
Appreciated 🙏